src/backend/distributed/commands/multi_copy.c

```diff
@@ -186,6 +190,187 @@ static void CopyAttributeOutText(CopyOutState outputState, char *string);
 static inline void CopyFlushOutput(CopyOutState outputState, char *start, char *pointer);
 
 
+bool
+WorkerCopy(CopyStmt *copyStatement)
+{
+	ListCell *optionCell = NULL;
+
+	/* Extract options from the statement node tree */
+	foreach(optionCell, copyStatement->options)
+	{
+		DefElem *defel = (DefElem *) lfirst(optionCell);
+
+		if (strncmp(defel->defname, "master_hostname", NAMEDATALEN) == 0)
```

Comment:
We don't use hostname very often. Typically nodename or "host" (libpq connection strings). Maybe master_host?